test.py

```diff
@@ -0,0 +1,71 @@
+import os
+import pytest
+from fastapi import FastAPI
+from fastapi.testclient import TestClient
+import psycopg2
+from psycopg2.extras import RealDictCursor
+from psycopg2.pool import ThreadedConnectionPool
+
+# Test app
+app = FastAPI()
```

Comment:
Rather than recreate the app, I would import the one from app.py. That way you don't have to duplicate the logic in this file